.azure-pipelines/azure-pipelines-build.yml

```diff
@@ -24,6 +24,7 @@ jobs:
       PLATFORM_AZP: $(GROUP_NAME)
       PLATFORM_ARCH: amd64
       BUILD_OPTIONS: ${{ parameters.buildOptions }}
+      DOCKER_DATA_ROOT_FOR_MULTIARCH: /data/multiarch/docker
```

Comment:
where is this variable getting used?

Reply:
azure-pipelines-image-template.yml:          export DOCKER_DATA_ROOT_FOR_MULTIARCH=/data/march/docker

Reply:
@lguohan , it is an the variable in Azure pipelines, and will be added as shell environment by azure pipelines' design. And Makefile.work will use it automatically.

Reply:
but there is already export in azure-pipeline-image-template.yml, why that does not work?

Reply:
We may need to set the variable for both configuration step and build image step.

Reply:
then, can you remove the export now? also use the original variable defined in the export /data/march/docker

Reply:
I removed the export, and use the old value /data/march/docker as the data root.